Resources/Prototypes/Catalog/VendingMachines/Inventories/gib.yml

```diff
@@ -8,6 +8,10 @@
     DrinkSolDryCan: 2
     DrinkLemonLimeCan: 2
     DrinkFourteenLokoCan: 2
+  contrabandInventory:
+    ClothingNeckStethoscope: 2
+    Saw: 2
+    Tourniquet: 3
```

Comment:
Not sure I understand why these are in the soda vendor.

Reply:
This is the Dr. Gibb machine. While most soda machines either receive a different soda (Dr. Gibb in PwrGame "Much Better Than Dr. Gibb" machines, or at least that's what they kept screaming at me during research and testing) or their full-sized bottle equivalents for the Cola/Soda and Space Mountain Wind machines, I wanted to try something more interesting for Dr. Gibb.

This is very much a field amputation kit, with a stethoscope for someone to throw on to pretend to be a doctor before doing it. While there is no amputation in the game, I wanted this to be somewhat jarring, that the seemingly innocent soda machine for some reason has these tools for amputation, with no anesthesia, wound dressing or anything else, and play into the shock value of seeing it for the first time. The reason to come back to this machine is the tourniquets, which is why their spawn is set to 3, so there's a higher chance that one will be there in a situation where you actually need it. 

tl;dr, field amputation kit in Dr. "tastes like malpractice" Gibb machine in horror game.